test/integration/parachain/staging/sequential/asset-registry.test.ts

```diff
@@ -48,10 +49,10 @@ describe("AssetRegistry", () => {
         if (newKeys.length > 0) {
             // clean up assets registered in test(s)
             const deleteKeysInstruction = interBtcAPI.api.tx.system.killStorage(newKeys);
-            await interBtcAPI.api.tx.sudo.sudo(deleteKeysInstruction).signAndSend(sudoAccount);
-
-            // wait for finalized event
-            await waitForIncludedEvent(interBtcAPI, api.events.sudo.Sudid);
+            await Promise.all([
+                waitForEvent(interBtcAPI, api.events.sudo.Sudid, false, approx10Blocks),
+                interBtcAPI.api.tx.sudo.sudo(deleteKeysInstruction).signAndSend(sudoAccount),
```

Comment:
> This usage of polkadot-js can be avoided by doing [something like this](https://github.com/interlay/interbtc-api/blob/ad8c3035a6ae36865399993d7c2eeefdb7cff214/test/integration/parachain/release/redeem.test.ts#L200). Doing this will also print all the events of this extrinsic to stdout.
Ah, good catch, thank you.
I believe I had that left over from a previous (failed) attempt to batch the delete calls.

Reply:
Tried to change it, but the signature of `sudo(InterBtcAPI, () =>  Promise<void>): Promise<void>` can not take `() => SubmittableExtrinsic<"prmoise", ISubmittableResult>` as second parameter.

Reply:
I see, the current `sudo` utility only works for txs already added to the lib interface, unlike `batch` for instance.